### PR TITLE
Fix dperecation warning on count.

### PR DESCRIPTION
**Pull request by dkinzer**
Fixes a bunch of deprecation warnings on this line when testing.

### CODE REVIEW

File: spec/features/home_facets_spec.rb [hunk 1]
```diff
@@ -6,10 +6,11 @@
 
   describe "Facets rendered" do
     it "Only a subset of all the facets render on the homepage" do
+      visit "/catalog?search_field=all_fields&q_1=x"
```
Comment (bibliotechy) [inline spec/features/home_facets_spec.rb]: What does this have to do with the deprecation?

Comment (dkinzer) [reply spec/features/home_facets_spec.rb]: Nothing, but I just started to randomly get an error on this test.  The search was pulling in 3 facets even though it should be 10.  I think because it was not done with the search (it's before I did all my reading on capybara docs).

Anyway, currently these tests seem to be brittle to timing issues.

Comment (dkinzer) [reply spec/features/home_facets_spec.rb]: btw, I was planning to do a PR to make this less brittle to timing problems after this gets merged, but I gess I can do that now and just reset this change... give me a sec.
